cmd/notary/integration_test.go

```diff
@@ -1526,3 +1532,374 @@ func TestWitness(t *testing.T) {
 		require.Error(t, err)
 	}
 }
+
+// Tests import/export keys
+func TestClientKeyImportExportOnly(t *testing.T) {
+	// -- setup --
+	setUp(t)
+
+	tempDir := tempDirWithConfig(t, "{}")
+	defer os.RemoveAll(tempDir)
+
+	server := setupServer()
+	defer server.Close()
+
+	var (
+		rootKeyID string
+	)
+
+	tempFile, err := ioutil.TempFile("", "pemfile")
+	require.NoError(t, err)
+	// close later, because we might need to write to it
+	defer os.Remove(tempFile.Name())
+
+	// -- tests --
+	// test 1, no path but role=root included with non-encrypted key
+	privKey, err := utils.GenerateECDSAKey(rand.Reader)
+	require.NoError(t, err)
+
+	pemBytes, err := utils.EncryptPrivateKey(privKey, "root", "", "")
```

Comment:
Non-blocking:  can we use `data.CanonicalRootRole` instead of "root" in this test here and a little further down?  Similarly with `data.CanonicalSnapshotRole` and "snapshot"?